NRasterizer.CLI/Program.cs

```diff
@@ -27,12 +27,17 @@ private static void BlitTo(Raster raster, Bitmap bitmap)
             {
                 bitmapData = b.LockBits(Entire(b), ImageLockMode.WriteOnly, b.PixelFormat);
                 int si = 0;
-                IntPtr di = bitmapData.Scan0;
-                for (int y = 0; y < b.Height; y++) 
+
+                unsafe
                 {
-                    Marshal.Copy(raster.Pixels, si, di, b.Width);
-                    si += raster.Stride;
-                    di += bitmapData.Stride;
+                    IntPtr di = bitmapData.Scan0;
+                    byte* di1 = (byte*)di;
+                    for (int y = 0; y < b.Height; y++)
+                    {
+                        Marshal.Copy(raster.Pixels, si, (IntPtr)di1, b.Width);
+                        si += raster.Stride;
+                        di1 += bitmapData.Stride; 
+                    }
```

Comment:
ah, right... :-/ hmm. Yeah, let's go with .net >= 4 for now then.

There are some tricks around it if we really need it.